static/js/components/EducationForm.js

```diff
@@ -66,30 +66,35 @@ class EducationForm extends ProfileFormFields {
   };
 
   renderEducationLevel(level){
-    const { ui: { educationDegreeInclusions }, profile: { education } } = this.props;
+    const {
+      ui: { educationDegreeInclusions },
+      profile: { education },
+      profile
+    } = this.props;
     if (educationDegreeInclusions[level.value]) {
       let rows = [];
       if (education !== undefined) {
         rows = Object.entries(education).
           filter(([, education]) => education.degree_name === level.value).
           map(([index, education]) => this.educationRow(education, index));
       }
-      rows.push(
-        <FABButton
-          colored
-          onClick={() => this.openNewEducationForm(level.value, null)}
-          className="profile-add-button"
-          key="I'm unique!"
-        >
-          <Icon name="add" />
-        </FABButton>
-      );
+      userPrivilegeCheck(profile, () => (
```

Comment:
Can you add braces here? Without the function braces it looks like we are using the return value of `rows.push` for something